packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-container.jsx

```diff
@@ -2,12 +2,12 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { Loading, useStateCallback } from '@deriv/components';
 import { WS } from '@deriv/shared';
-import Expired from 'Components/poa/status/expired';
-import Unverified from 'Components/poa/status/unverified';
-import NeedsReview from 'Components/poa/status/needs-review';
-import Submitted from 'Components/poa/status/submitted';
-import Verified from 'Components/poa/status/verified';
-import NotRequired from 'Components/poa/status/not-required';
+import PoaExpired from 'Components/poa/status/expired';
+import PoaUnverified from 'Components/poa/status/unverified';
+import PoaNeedsReview from 'Components/poa/status/needs-review';
+import PoaSubmitted from 'Components/poa/status/submitted';
```

Comment:
just renamed for improving readability. It is supposed, that similar components will be used in proof of income feature. They will be named accordingly